packages/Toast/stories/Toast.examples.stories.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import Toast from "../src";
+import ControlledToastExample from "./examples/ControlledToastExample";
+import FixedToastExample from "./examples/FixedToastExample";
+import UncontrolledToastExample from "./examples/UncontrolledToastExample";
+
+export default {
+  title: "Toast/Examples",
+  component: Toast,
+};
+
+export const controlledToast = () => <ControlledToastExample />;
+controlledToast.story = {
+  parameters: {
+    options: {
+      isToolshown: true,
+      showPanel: true,
+    },
+  },
+};
```

Comment:
Since these options are all the same, there's probably an opportunity to DRY this up a bit.